CMakeLists.txt

```
@@ -58,6 +62,7 @@ set(CMAKE_CXX_FLAGS ${YARP_CXX_FLAGS})
 
 #### INSERT find_package(MATIO) and other libraries here
 find_package(matioCpp REQUIRED)
+find_package(Boost REQUIRED COMPONENTS filesystem system)
```

If you only use a header-only library, I think we can just do:
```suggestion
find_package(Boost REQUIRED)
```